tests/unit/SchemaForm.spec.js

```diff
@@ -36,6 +36,57 @@ describe('SchemaForm', () => {
     expect(wrapper.findAllComponents(FormText)).toHaveLength(2)
   })
 
+  describe('array schema', () => {
```

Comment:
the first two tests below seem to do the same thing, so perhaps having the second one only (nested arrays) is sufficient